### ISSUE TITLE
Img element missing alt text

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#703]
# a11y bug
```
Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #main > header > div:nth-child(2) > div > div:nth-child(1) > a
   └── <a href="#" data-activate-element="" data-activate-noscroll="true" data-activeid="offcanvasMenu"><img class="Header-hamb__icon" ...</a>
```
Experienced in /it/18app page.
Please note: this is not a single page related bug but most probably the fix has to be provided in the layout that the page inherits.

**Comment by 4Rebin** [comment]
I would like to work on this issue.

**Comment by Trustidiong** [comment]
Hey, I would like to work on this issue. I think it will be fun.

**Comment by soulhunter10737** [comment]
I would like to take this

**Comment by libremente** [comment]
Hi @Trustidiong, thanks for your interest! Waiting for you PR!

**Comment by bfabio** [comment]
Hello @Trustidiong, are you still working on this?
